src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/DefaultMotion.tsx

```diff
@@ -19,32 +19,33 @@ interface DefaultMotionProps {
   actionData: ColonyAction;
 }
 
-const DefaultMotion = ({ actionData }: DefaultMotionProps) => {
-  const { colony } = useColonyContext();
+const DefaultMotion = ({
+  actionData,
+  actionData: { motionData },
+}: DefaultMotionProps) => {
+  const { colony, showBanner, setShowBanner, showMotionHeading } =
+    useDefaultMotion(motionData);
```

Comment:
I appreciate your efforts to keep the components lean, but I can't see a benefit in this particular hook. One of the reasons might be that it's primary purpose is to encapsulate some of this component dependencies and I don't think this is worthwhile.
Apart from what Raul mentioned regarding exporting the colony, I think exporting information on whether or not to show motion heading makes this hook more or less useless. It's not great in terms of reusability (maybe another component has got a different rules on when to show the heading) and it violates the separation of concerns between the data and the view that I think you originally wanted to improve. 
A good rule is that a piece of code should only have one reason to change, and while it's not always possible, I'd think a good reason for this hook to change would be some changes to the actions or extensions data. But right now, the hook also has to change whenever the underlying UI changes, meaning we will most likely be unable to use it for the refreshed UI "out of the box". 

Maybe this refactoring/optimisation happens a bit too early? This component was only ~90 lines long which I'd normally consider pretty short and there wasn't a ton of complex logic that you got rid of. I can imagine there's gonna be some more motions data mapping/manipulation down the line and it might be easier to see its value then